### ISSUE TITLE
Crashes imediately unless sudo (Arch Linux)

### ISSUE THREAD

**Issue by gwemmie** [haskellcamargo/skype-unofficial-client#42]
I just installed the AUR package, and when I launch the program either through /opt/skype_unofficial_client/skype or /usr/bin/skype-desktop, it flashes the window and the Skype logo and crashes before reaching the login screen. There is no output related to the crash; only one line: "[0302/090355:INFO:nw_package.cc(175)] /opt/skype_unofficial_client/skype". After some fiddling, I figured out that it works fine if I run it with sudo. I've tried chmodding and chowning /opt/skype_unofficial_client to be writable by me, but neither of those methods work. I simply have to run it as root.

(gksu also works)

**Comment by gwemmie** [comment]
I found a new aspect to this behavior. Not only does it not launch properly unless I launch it as root, but it also inexplicably doesnt launch through alltray (as root) unless I first launch it NORMALLY as root (without alltray), log in, close it, and THEN use alltray. I have no idea why, but it's extremely annoying, since an app like this is useless without a tray icon.

**Comment by gwemmie** [comment]
OK, now I can't actually get alltray to work with it at all. I'll just open that as a separate issue then.

**Comment by closingin** [comment]
I have the same problem (the client opens a window and exits) but it seems like running the program with sudo doesn't help. I get the same output : 

`closingin@laptop1-linux1: skype-desktop`
`[0330/225835:INFO:nw_package.cc(175)] ./skype --enable-spell-checking`

Is there any option to enable debug or fix this ? By the way, by launching the app with no internet connection, i can get the window to stay opened, telling me i have no internet connection.

**Comment by gizmocodes** [comment]
I can say that the package from AUR works fine for me. What AUR Helper are you using (For example, I use Aura, while most people use Packer or Yaourt). Or are you using a helper?

**Comment by closingin** [comment]
I'm not using any, I download the tarball and use makepkg.

**Comment by BernardoGO** [comment]
This is happening for me too.
Tried to install it from AUR using yaourt and also tried to compile it, same problem.

**Comment by nikita2206** [comment]
It also dumps the core by the way:

```
Process 19202 (skype) of user 1000 dumped core.

Stack trace of thread 19202:
#0  0x00007f95ac253977 n/a (libfreetype.so.6)
#1  0x00007f95ac254367 n/a (libfreetype.so.6)
#2  0x00007f95ac251c8f n/a (libfreetype.so.6)
#3  0x00007f95ac207cb1 FT_Load_Glyph (libfreetype.so.6)
#4  0x00007f95ac208070 FT_Get_Advances (libfreetype.so.6)
#5  0x00007f95b05d4ffc n/a (libnw.so)
```

**Comment by nikita2206** [comment]
By the way, I happen to have `Infinality-bundle` installed with patched freetype2 (infinality-freetype2), seems like it's causing the issue

**Comment by HighIO** [comment]
@nikita2206   Hi, it works fine with `infinality-bundle` in my Arch except the error
`[8288:8288:0702/183855:ERROR:browser_main_loop.cc(253)] Gtk: IA__gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed`

Besides I had to `setfattr -n user.pax.flags -v em /opt/skype_unofficial_client/skype` in linux-grsec kernel

**Comment by Kuchiriel** [comment]
This issue persists?

**Comment by scorpp** [comment]
yes, happens to me to. Archlinux, built with makepkg, freetype2 from `infinality-bundle`

```
Stack trace of thread 24334:
               #0  0x00007efe4c944c54 n/a (libfreetype.so.6)
               #1  0x00007efe4c945da0 n/a (libfreetype.so.6)
               #2  0x00007efe4c94420f n/a (libfreetype.so.6)
               #3  0x00007efe4c8fa1f1 FT_Load_Glyph (libfreetype.so.6)
               #4  0x00007efe4c8fa5b0 FT_Get_Advances (libfreetype.so.6)
               #5  0x00007efe50cdbffc n/a (libnw.so)
```

Update: i installed nwjs-bin from AUR and started skype-unofficial with `nw .` and it worked fine.
